man/fi_endpoint.3.md

```diff
@@ -187,7 +187,12 @@ bound to one or more fabric resources.  An endpoint that will generate
 asynchronous completions, either through data transfer operations or
 communication establishment events, must be bound to the appropriate
 completion queues or event queues, respectively, before being enabled.
-Unconnected endpoints must be bound to an address vector.
+Additionally, endpoints that use manual progress must be associated
+with relevant completion queues or event queues in order to drive
+progress.  For endpoints that are only used as the target of RMA or
+atomic operations, this means binding the endpoint to a completion
+queue associated with receive processing.  Unconnected endpoints must
```

Comment:
Binding a counter should also work for the purpose of making manual progress.

Reply:
I thought about this.  We don't allow an endpoint to be bound to a counter without also being bound to a completion queue on the initiator side.  I'm not sure we want to allow that case at the target side.  The current text for binding to a counter says, "An endpoint may also be bound to a fabric counter", though that was a recent clarification.